### PR TITLE
Add a react-admin generator

### PR DESCRIPTION
**Pull request by jvasseur**
Fix https://github.com/thetribeio/generator-project/issues/338

Usage:  by responding yes in the app generator or using `yo @thetribe/project:react-admin admin --http-path=/admin/`

TODO:
- [x] Add a question to app generator to allow generating the admin.
- [x] Remove index.css
- [x] Remove logo.svg
- [ ] Update backend to add required API ?
- [x] Add auth provider

### CODE REVIEW

File: generators/react-admin/templates/base/src/authProvider.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { AuthProvider } from 'react-admin';
+
+// TODO: Update auth provider to match your API authentification
+const authProvider = (): AuthProvider => ({
+    checkAuth: async () => {},
+    checkError: async () => {},
+    getPermissions: async () => {},
+    login: async (params) => {
+        await fetch('/api/logout', {
```
Comment (qboisson) [inline generators/react-admin/templates/base/src/authProvider.ts:9]: login

Comment (jvasseur) [reply generators/react-admin/templates/base/src/authProvider.ts]: Fixed

**Review by blatinier — APPROVED** [review]
I'm not knowledgable enough in react-admin to provide meaningful insight but the rest looks good 👌

**Comment by jvasseur** [comment]
@Maeevick it's true that some packages could be updated, but in the case of Jest, it's CRA that's not compatible with the v27

**Pull request merged**